### PR TITLE
Change most remaining ostream operator<< to fmt::format_to

### PR DESCRIPTION
**Pull request by scamille**
Use util/format.hpp for easier implementation.

### CODE REVIEW

File: engine/util/timespan.cpp [hunk 1]
```diff
@@ -3,15 +3,13 @@
 // Send questions to natehieter@gmail.com
 // ==========================================================================
 
-#include "sc_timespan.hpp"
-
 #include "fmt/format.h"
-#include "fmt/ostream.h"
+#include "sc_timespan.hpp"
```
Comment (nuoHep) [inline engine/util/timespan.cpp]: Leave the order as-is?

**Review by nuoHep — APPROVED** [review]
LGTM overall

File: engine/util/timespan.cpp (hunk 1 above)
Comment (scamille) [reply engine/util/timespan.cpp]: that was clang-format, because the files have different names. Will fix.

**Comment by scamille** [comment]
I thought it would be a good time to move sc_timespan.hpp :-)

**Pull request merged**